packages/hardhat/scripts/listAccount.ts

```diff
@@ -3,17 +3,26 @@ dotenv.config();
 import { ethers, Wallet } from "ethers";
 import QRCode from "qrcode";
 import { config } from "hardhat";
+import password from "@inquirer/password";
 
 async function main() {
-  const privateKey = process.env.DEPLOYER_PRIVATE_KEY;
+  const encryptedKey = process.env.DEPLOYER_PRIVATE_KEY_ENCRYPTED;
```

Comment:
We could actually list the account without decrypting, since in generate JSON from ethers (where the PK is encrypted) you can get the Ethereum address.

But decided you keep it like this... in case someone forgets their password and send ETH directly.

Reply:
> But decided you keep it like this... in case someone forgets their password and send ETH directly

I don't get what you mean here. If someone forgets the password, can he see the balances?

Reply:
If you forget your password:

- you run `yarn account` and it'll ask for your password. So you won't send anything
- If we don't require a password, you might send it... and then (e.g. when you deploy) you'd realize that you don't remember the password and the funds might be lost.

That was the only reason I did it like this.

Happy to reconsider.

Reply:
> But decided you keep it like this... in case someone forgets their password and send ETH directly.

Yeah makes sense! and in foundry also we have same worksflow

Reply:
> That was the only reason I did it like this.

Got it, thanks!